auth.py

```diff
@@ -1,31 +1,70 @@
 import authx.auth
 import os
 import re
+import json
 
+import requests
 
-AUTH = True
 
+AUTH = True
 
-def get_auth_header():
+def get_auth_header(refresh_token=None):
     if AUTH:
         import auth
-        token = auth.get_site_admin_token()
-        return {"Authorization": f"Bearer {token}"}
+        if refresh_token:
+            refresh_token = auth.get_refresh_token(refresh_token)
+        else:
+            refresh_token = auth.get_site_admin_token()
+        token = get_bearer_from_refresh(refresh_token)
+        return {"Authorization": f"Bearer {token}", "refresh_token": refresh_token}
     return ""
 
 
-def get_site_admin_token():
-    return authx.auth.get_access_token(
-    keycloak_url=os.getenv('KEYCLOAK_PUBLIC_URL'),
-    client_id=os.getenv('CANDIG_CLIENT_ID'),
-    client_secret=os.getenv('CANDIG_CLIENT_SECRET'),
-    username=os.getenv('CANDIG_SITE_ADMIN_USER'),
-    password=os.getenv('CANDIG_SITE_ADMIN_PASSWORD')
-    )
+def get_site_admin_token(refresh_token=None):
+    if not refresh_token:
+        return authx.auth.get_refresh_token(
+        keycloak_url=os.getenv('KEYCLOAK_PUBLIC_URL'),
+        client_id=os.getenv('CANDIG_CLIENT_ID'),
+        client_secret=os.getenv('CANDIG_CLIENT_SECRET'),
+        username=os.getenv('CANDIG_SITE_ADMIN_USER'),
+        password=os.getenv('CANDIG_SITE_ADMIN_PASSWORD')
+        )
```

Comment:
This doesn't work. I had to switch back to the old `authx.auth.get_access_token`. Modifying the `get_site_admin_token`  while also refreshing the token is confusing and could have unexpected behavior. Suggest to keep it and have a separate refresh token function

Reply:
Purpose of auth methods clarified in [892ec1a](https://github.com/CanDIG/candigv2-ingest/pull/18/commits/892ec1aa1ff6ab1e084011ef861f36f90f3dd8d2)